lib/plugins/output-filter/kubernetes-enrichment.js

```diff
@@ -0,0 +1,212 @@
+const { KubeConfig } = require('kubernetes-client')
+const Client = require('kubernetes-client').Client
+const Request = require('kubernetes-client/backends/request')
+const kubeconfig = new KubeConfig()
+var consoleLogger = require('../../util/logger.js')
+var podCache = {}
+var digestRegEx = /sha256:.+/i
+var client = null
+const FALSE_REGEX = /false/i
+if (process.env.KUBERNETES_PORT_443_TCP !== undefined) {
+  kubeconfig.loadFromCluster()
+} else {
+  kubeconfig.loadFromDefault()
+}
+client = new Client({ backend: new Request({ kubeconfig }) })
+client.loadSpec().then(() => {}).catch(error => {
```

Comment:
Why do you have an empty `.then(() => {})`? You can just directly go to the `.catch()` without any `.then()` if you're only looking for handling errors.

Reply:
ok, will remove it. Maybe a leftover from some initial debug output, just reporting when k8s API is initialized.